pom.xml

```diff
@@ -374,6 +377,7 @@
                     </execution>
                 </executions>
             </plugin>
+            <!--
             <plugin>
                 <groupId>com.exasol</groupId>
                 <artifactId>error-code-crawler-maven-plugin</artifactId>
```

Comment:
There was some issues because of Java 8 class crawling, I did not want to investigate it at the moment, I'll check again later. Going to remove the comment.

Reply:
Created #18 not to forget it.

Reply:
Please create a ticket to activate the plugin again later